cernopendata_client/searcher.py

```diff
@@ -16,6 +16,7 @@
 
 import click
 import requests
+from .config import SERVER
```

Comment:
I believe having a blank line before the dot import is the usual convention.

Reply:
Done 👍